src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -449,7 +449,13 @@ export default (accessToken, userID, opts) => {
       {},
       { method: "PUT" }
     ),
-    meUserInterestsLoader: gravityLoader("me/user_interests"),
+    meUserInterestsLoader: gravityLoader(
+      "me/user_interests",
+      {},
+      {
+        headers: true,
+      }
```

Comment:
We need the headers to get the `totalCount`. I've adjusted all resolvers using this loader.